pkg/util/winutil/go.mod

```diff
@@ -0,0 +1,23 @@
+module github.com/DataDog/datadog-agent/pkg/util/winutil
+
+go 1.14
+
+replace github.com/DataDog/datadog-agent/pkg/util/log => ../log
+
+require (
+	github.com/DataDog/datadog-agent/pkg/util/log v0.0.0
+	github.com/client9/misspell v0.3.4
+	github.com/frapposelli/wwhrd v0.2.4
+	github.com/fzipp/gocyclo v0.3.1
+	github.com/golangci/golangci-lint v1.27.0
+	github.com/gordonklaus/ineffassign v0.0.0-20210103220932-664217a59c00
+	github.com/mailru/easyjson v0.7.6
+	github.com/shuLhan/go-bindata v3.6.1+incompatible
+	github.com/stretchr/testify v1.6.1
```

Comment:
I'm a bit curious about some of these - they look like they're in `tools.go` too. I would expect that there would be 0 overlap between them. May be good to check the same for other modules that are created with this PR.

Reply:
`tools.go` packages will end up on the `go.mod` file of the module since they are technically dependencies. #7750 should fix this (it moves `tools.go` to its own module, so it won't pollute the `go.mod` files)